apps/tutorial-servo-can-control/src/main.cpp

```diff
@@ -0,0 +1,25 @@
+#include "TutorialServo.h"
+#include "CANMsg.h"
+// mbed.h already included in TutorialServo.h
+
+/*
+CAN packet structure for servo
+data 4-byte float put in via << overload
+float represents desired angle
+*/
+
+Ticker ticker;
+CAN can1(p9, p10);

```

Comment:
For CAN you should use the `CAN_RX` and `CAN_TX` pins. Also idk if it's required for training, but FW uses `CANBus` instead of `CAN` - can @younesr1 confirm?